### PR TITLE
Fix up for pre-trained checkpoints and their datasets

### PR DESCRIPTION
**Pull request by dan-wells**
This all works now to synthesize from ming024's pre-trained models, and to train LJ from scratch. I haven't tested LibriTTS or AISHELL3, but they should be consistent here.

I added some more explicit notes throughout the README which I hope should preempt all the errors I ran into while I was testing this.

I haven't tried out the whole process of adding a new language myself, but I've added more explicit pointers to code locations for each of the additions listed in that section of the README. I hope this should put people close enough to the existing files or code in each case that they can pick up the examples and run with them.

### CODE REVIEW

File: text/symbols.py [hunk 1]
```diff
@@ -26,12 +26,12 @@
 _moh_punctuation = "!(),.;? "
 _special = "-"
 _letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
-_silences = ["@sp", "@spn", "@sil"] # ["sp", "spn", "sil", "@sp", "@spn", "@sil"] can be used as well
+_silences = ["sp", "spn", "sil"] # ["sp", "spn", "sil", "@sp", "@spn", "@sil"] can be used as well
```
Comment (dan-wells) [inline text/symbols.py:29]: This matches (at least) the short pause tokens in ming024's MFA TextGrids. Using `@sp` etc. lead to text/pitch sequence length mismatches when training a new LJ model.

File: requirements.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ nltk==3.5
 numba == 0.48
 numpy == 1.19.0
 panphon==0.19
+pillow==9.1.0
```
Comment (dan-wells) [inline requirements.txt:11]: For TensorBoard spectrogram plots.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ This fork of ming024's implementation was used for my M.Sc. Speech & Language Pr
 # Changes from ming024
 - Model allows removal of energy variance adaptor, postnet and optional depthwise separable convolutions similar to LightSpeech (Luo et. al. 2021). Changes can be made in `config/*/model.yaml`
 - Model allows for one-hot learnable language embeddings. Changes can be made in `config/*/model.yaml`
-- Model allows for deep-speaker based speaker embeddings (in addition to one-hot implementation from ming024). Changes can be made in `config/*/model.yaml`
+- Model allows for deep-speaker based speaker embeddings (in addition to one-hot implementation from ming024). Clone with `--recurse-submodule` or run `git submodule update --init` after cloning to enable. Changes can be made in `config/*/model.yaml`
```
Comment (roedoejet) [inline README.md:8]: good note - thanks!

**Review by roedoejet — APPROVED** [review]
Thanks so much @dan-wells !

**Pull request merged**